src/store/dappChainStore.js

```diff
@@ -30,8 +30,8 @@ if (hostname === "dashboard.dappchains.com") {
   LOOM_ADDRESS = "0x165245382ff23A5D3782b48286B6A81b6fd0508e"
   GW_ADDRESS = "0x76c41eFFc2871e73F42b2EAe5eaf8Efe50bDBF73"
 } else {
-  LOOM_ADDRESS = ""
-  GW_ADDRESS = ""
+  LOOM_ADDRESS = "0x165245382ff23A5D3782b48286B6A81b6fd0508e"
+  GW_ADDRESS = "0x76c41eFFc2871e73F42b2EAe5eaf8Efe50bDBF73"
```

Comment:
this gateway change won't affect the vm right?

Reply:
Oh that change slipped in. I'll revert it